internal/server/handlers_test.go

```diff
@@ -990,7 +1022,8 @@ func TestAPI_GetUser(t *testing.T) {
 					  "name": "me@example.com",
 					  "lastSeenAt": "%[2]v",
 					  "created": "%[2]v",
-					  "updated": "%[2]v"
+					  "updated": "%[2]v",
+						"providerNames": null
```

Comment:
nit, this formatting is off
```suggestion
					"providerNames": null
```